### PR TITLE
take active blobbers time limit from sc.yaml

### PR DESCRIPTION
**Pull request by boddumanohar**
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Review by dabasov — APPROVED** [review]
lgtm

**Review by dabasov — COMMENTED** [review]
delete all the references to ActiveBlobbersTimeLimit

File: code/go/0chain.net/smartcontract/storagesc/handler.go [hunk 1]
```diff
@@ -1279,7 +1280,12 @@ func (srh *StorageRestHandler) validators(w http.ResponseWriter, r *http.Request
 	var validators []event.Validator
 
 	if active == "true" {
-		validators, err = edb.GetActiveValidators(pagination)
+		conf, err2 := getConfig(srh.GetQueryStateContext())
+		if err2 != nil && err2 != util.ErrValueNotPresent {
+			common.Respond(w, r, nil, smartcontract.NewErrNoResourceOrErrInternal(err2, true, cantGetConfigErrMsg))
+			return
+		}
+		validators, err = edb.GetActiveValidators(pagination, conf.HealthCheckPeriod)
```
Comment (peterlimg) [inline code/go/0chain.net/smartcontract/storagesc/handler.go]: The `conf` can be nil if we get `util.ErrValueNotPresent` error, so let's do a nil check before using it.

File: code/go/0chain.net/smartcontract/storagesc/handler.go [hunk 2]
```diff
@@ -2196,7 +2202,12 @@ func (srh *StorageRestHandler) getBlobbers(w http.ResponseWriter, r *http.Reques
 
 	var blobbers []event.Blobber
 	if active == "true" {
-		blobbers, err = edb.GetActiveBlobbers(limit)
+		conf, err2 := getConfig(srh.GetQueryStateContext())
+		if err2 != nil && err2 != util.ErrValueNotPresent {
+			common.Respond(w, r, nil, smartcontract.NewErrNoResourceOrErrInternal(err2, true, cantGetConfigErrMsg))
+			return
+		}
+		blobbers, err = edb.GetActiveBlobbers(limit, conf.HealthCheckPeriod)
```
Comment (peterlimg) [inline code/go/0chain.net/smartcontract/storagesc/handler.go]: Same as above

**Review by peterlimg — DISMISSED** [review]

File: code/go/0chain.net/smartcontract/storagesc/handler.go (hunk 1 above)
Comment (boddumanohar) [reply code/go/0chain.net/smartcontract/storagesc/handler.go]: done

File: code/go/0chain.net/smartcontract/storagesc/handler.go (hunk 2 above)
Comment (boddumanohar) [reply code/go/0chain.net/smartcontract/storagesc/handler.go]: done

**Review by dabasov — APPROVED** [review]
lgtm

**Pull request merged**